_unittest/test_27_Maxwell2D.py

```diff
@@ -108,3 +108,13 @@ def test_16_enable_inductance_computation(self):
         assert self.aedtapp.change_inductance_computation()
         assert self.aedtapp.change_inductance_computation(True, False)
         assert self.aedtapp.change_inductance_computation(False, False)
+
+    # @pyaedt_unittest_check_desktop_error
+    # def test_17_enable_inductance_computation(self):
+    #     # Set a new value for model depth
+    #     self.aedtapp.model_depth = "90 mm"
+    #     assert self.aedtapp.model_depth == 0.09
+    #     self.aedtapp.model_depth = "1.3"
+    #     assert self.aedtapp.model_depth == 1.3
+    #     # Generate design data
+    #     assert self.aedtapp.generate_design_data()
```

Comment:
If not needed we can remove this comment

Reply:
Done. I will reinstate the test when the project properties issue is fixed.